kusto-language-server/client/src/extension.ts

```diff
@@ -105,6 +105,16 @@ commands.registerCommand('kuskus.loadSymbols', async () => {
 			return;
 		}
 
+		const authId = await window.showInputBox({
+			ignoreFocusOut: true,
+			placeHolder: 'common',
+			prompt: 'Authority/Tenant ID'
```

Comment:
Is there some way to provide a bit more info to the user on what "authority/tenant ID" means and how they'd find out what they should use? E.g. is there another param to `.showInputBox` like `hint` or similar? I wouldn't know what to do here

Reply:
The tenant id is available in the Azure Portal on Azure Active Directory home page. I'll do some brainstorming to see how to best convey this info to potential users.

Reply:
functionality i would really like to add would be something like in the [vscode-cosmosdb](https://github.com/Microsoft/vscode-cosmosdb) extension. If you scroll down on the readme, Will Lorey would be the readable name of the tenant id number and then you can see a list of databases, expand to get a list of tables, etc...

Reply:
Nice yeah I think that would be a good future contribution.